File: Angular/satva/src/app/feedback/feedback.ts
export class Feedback {
    private custConsumerId: number = 0;
    public get custConsumerid(): number {
        return this.custConsumerId;
    }
    public set custConsumerid(value: number) {
        this.custConsumerId = value;
    }
	private rating: number = 0;
    public get rate(): number {
        return this.rating;
    }
    public set rate(value: number) {
        this.rating = value;
    }
	private suggestions: string = '';
    public get suggest(): string {
        return this.suggestions;
    }
    public set suggest(value: string) {
        this.suggestions = value;
    }
}

File: SpringBoot/demo/src/main/java/com/example/rps/beans/Feedback.java
package com.example.rps.beans;

public class Feedback 
{
	private int custConsumerId;
	private int rating;
	private String suggestions;
	public int getCustConsumerId() {
		return custConsumerId;
	}
	public void setCustConsumerId(int custConsumerId) {
		this.custConsumerId = custConsumerId;
	}
	public int getRating() {
		return rating;
	}
	public void setRating(int rating) {
		this.rating = rating;
	}
	public String getSuggestions() {
		return suggestions;
	}
	public void setSuggestions(String suggestions) {
		this.suggestions = suggestions;
	}

}
